### PR TITLE
Feature/update toil

### PR DESCRIPTION
**Pull request by nikhil**

### CODE REVIEW

**Comment by nikhil** [comment]
This needs to be tested on all the pipelines first in staging

File: submitter/toil_submitter/toil_jobsubmitter.py [hunk 1]
```diff
@@ -227,6 +227,8 @@ def _command_line(self):
                 "--disable-host-provenance",
                 "--stats",
                 "--debug",
+                "--disableProgress",
+                "--doubleMem",
```
Comment (ionox0) [inline submitter/toil_submitter/toil_jobsubmitter.py:231]: We might also consider making use of the `--statePollingWait` parameter here

**Review by ionox0 — COMMENTED** [review]
This is going to be a helpful change for us, since we are seeing an error about the temp file names being too long, and I think this was fixed in this new toil version

**Comment by ionox0** [comment]
Let me know when this is in staging and I will test the Access pipelines

File: submitter/toil_submitter/toil_jobsubmitter.py (hunk 1 above)
Comment (nikhil) [reply submitter/toil_submitter/toil_jobsubmitter.py:231]: Good idea!

**Comment by nikhil** [comment]
Hmmm, the tests worked locally. I will go ahead with the staging tests, and see if I can fix this after.

**Comment by nikhil** [comment]
Tests passed, but coverage failed

File: ridgeback/settings.py [hunk 2]
```diff
@@ -243,7 +243,8 @@
 TOIL_TMP_DIR_ROOT = os.environ["RIDGEBACK_TOIL_TMP_DIR_ROOT"]
 LSF_WALLTIME = os.environ["RIDGEBACK_LSF_WALLTIME"]
 LSF_SLA = os.environ["RIDGEBACK_LSF_SLA"]
-CWLTOIL = os.environ.get("RIDGEBACK_TOIL", "cwltoil")
+CWLTOIL = os.environ.get("RIDGEBACK_TOIL", "toil-cwl-runner")
+TOIL_STATE_POLLING_WAIT = os.environ.get("TOIL_STATE_POLLING_WAIT",60)
```
[the remark this answers is not in the document]
Comment (nikhil) [reply ridgeback/settings.py]: Ahh, thanks!

**Review by ionox0 — APPROVED** [review]
I've successfully tested our nucleo operator using this PR

**Review by allanbolipata — APPROVED** [review]

**Pull request merged**